test/test_models.py

```diff
@@ -148,7 +148,20 @@ def _test_detection_model(self, name, dev):
         kwargs = {}
         if "retinanet" in name:
             kwargs["score_thresh"] = 0.013
-        model = models.detection.__dict__[name](num_classes=50, pretrained_backbone=False, **kwargs)
+
+        # Workaround for flaky tests
+        from torchvision.ops.misc import FrozenBatchNorm2d
+
+        class OverwriteEPS:
+            def __enter__(self):
+                self.default = FrozenBatchNorm2d._DEFAULT_EPS
+                FrozenBatchNorm2d._DEFAULT_EPS = 0.0
+
+            def __exit__(self, type, value, traceback):  # noqa: F811
+                FrozenBatchNorm2d._DEFAULT_EPS = self.default
+
+        with OverwriteEPS():
+            model = models.detection.__dict__[name](num_classes=50, pretrained_backbone=False, **kwargs)
```

Comment:
Instead of doing this workaround (and having to introduce test-specific code in the op implementation, what about replacing all occurrences of `FrozenBatchNorm.eps` with the value 0?

Something like
```python
model = models.detection.__dict__[name](num_classes=50, pretrained_backbone=False, **kwargs)
for module in model.modules():
    if isinstance(module, FrozenBatchNorm2d):
        module.eps = 0
```

Reply:
I like this better than I like mine workaround. Still we are putting lipstick on a pig but it will work for now. :)